### PR TITLE
[WIP] initial add of Rahman et al. reference

### PR DESCRIPTION
**Pull request by ctb**

### CODE REVIEW

File: content/04.discussion.md [hunk 1]
```diff
@@ -27,6 +27,8 @@ used efficiently and effectively for compositional analysis of shotgun
 metagenome data sets with k-mers. In particular, FracMinHash is
 competitive in accuracy with extant MinHash-based techniques for
 containment analysis, while also supporting Jaccard similarity.
+Further analytical results for FracMinHash are available in
```
Comment (dkoslicki) [inline content/04.discussion.md:30]: In addition, FracMinHash can be used to obtain point estimates of and confidence intervals around mutation rates and evolutionary distances, see [@dk_fracminhash] for these and other analytical results.

File: content/03.results.md [hunk 2]
```diff
@@ -38,7 +38,9 @@ of the containment index:
 ```
 {% endraw %}
 
-See Methods for details.
+See Methods for details. Detailed analysis of bias and confidence
```
Comment (dkoslicki) [inline content/03.results.md:41]: I think this sentence is not in the best place. Perhaps one paragraph down, after "This permits a variety of convenient operations on the sketches, including iterative downsampling of FracMinHash sketches as well as conversion to MinHash sketches." 
One could add 
"Beyond accurate containment operations, FracMinHash can be used to estimate evolutionary distance between pairs of sequences undergoing a mutation model, similar to but more accurately than the MinHash derived method in [@mash]. See [@dk_fracminhash] for these and other analytical details."

**Review by dkoslicki — CHANGES_REQUESTED** [review]

**Comment by dkoslicki** [comment]
I've expanded on this and suggested a moving of a sentence. All suggestions of course

File: content/04.discussion.md (hunk 1 above)
Comment (ctb) [reply content/04.discussion.md:30]: updated in 5603929

File: content/03.results.md (hunk 2 above)
Comment (ctb) [reply content/03.results.md:41]: updated in 56039295ecffb3bd6485efe099530bfc7393eeea

**Pull request merged**